src/index.html

```diff
@@ -20,8 +20,396 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="container">
+      <a
+        class="logo"
+        href="#"
+      >
+        <img
+          class="logo__image"
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li
+            class="nav__item"
+            href="#"
+          >
+            <a class="is-active nav__link">Apple</a>
+          </li>
+          <li
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li
```